app/services/analytics_events.rb

```diff
@@ -628,19 +631,28 @@ def idv_inherited_proofing_redo_retrieve_user_info_submitted(**extra)
   end
 
   # @param [String] flow_path Document capture path ("hybrid" or "standard")
+  # @param [String] in_person_cta_variant Variant testing bucket label
   # The user visited the in person proofing location step
-  def idv_in_person_location_visited(flow_path:, **extra)
-    track_event('IdV: in person proofing location visited', flow_path: flow_path, **extra)
+  def idv_in_person_location_visited(flow_path:, in_person_cta_variant:, **extra)
```

Comment:
it looks like the `${stepName} visited` steps only sometimes send `in_person_cta_variant` so does it need to be nillable?
```suggestion
  def idv_in_person_location_visited(flow_path:, in_person_cta_variant: nil, **extra)
```
(same for other methods too)

Reply:
I believe the intent is to include the value at all times for these events. Am I forgetting a state as I think through this?

Reply:
Yeah I meant looking at the `else` case of this part of the code, it looks like there *could* be cases where that's not sent?

https://github.com/18F/identity-idp/pull/7806/files#r1110036395

Reply:
Gotcha! This method is specifically called in the `location` case. The else matches other `AnalyticsEvent` methods.